### PR TITLE
Windows simple script

### PR DESCRIPTION
**Pull request by megafon929**

### CODE REVIEW

**Comment by Prince90446** [comment]
plzzzz add mt6779

**Comment by hoangtrongfb** [comment]
mt6853 add please

**Comment by eminemgsm** [comment]
Plz add MT6779

**Comment by chaosmaster** [comment]
This seems kind of silly

File: brom.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@echo off
+python main.py
```
Comment (W1NN3R) [inline brom.bat:2]: it doesnt run  and gives error ! 
while all drivers is good

Comment (megafon929) [reply brom.bat:2]: It works for everyone, but you don't, very strange

**Comment by Slacoblochina** [comment]
Can you please add helio g96 support. Because the android utility requires a smart card, which I don't have.

**Pull request closed without merging**